### PR TITLE
Add list of supported mission commands

### PR DESCRIPTION
**Pull request by hamishwillee**
@bkueng As discussed :-). List of mission commands that are supported. 
- I put this in the Mission Mode doc as this is a more "technical" description of what is supported by the mode.
- Has caveats at the end, including that it might get out of date (and how to check)
- I have assumed that things work as per spec. I noted one case (fly through) where this is not the case. If you know any others, let me know!
- I noted one frame type that I believe is not supported. Let me know if you know of others.

I also updated the basic QGC "Flying Missions" doc just to bring it to an "OK" standard and link to better docs on QGC.

### CODE REVIEW

**Comment by hamishwillee** [comment]
@bkueng According to https://github.com/PX4/Firmware/issues/9974 DO_REPOSITION works, but it is not listed in the handler. Am I missing something?

File: en/flight_modes/mission.md [hunk 1]
```diff
@@ -49,6 +54,69 @@ For more information about mission planning, see:
...
+* [MAV_CMD_DO_VTOL_TRANSITION](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_VTOL_TRANSITION)
+* [MAV_CMD_NAV_DELAY](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_DELAY)
+* [MAV_CMD_NAV_RETURN_TO_LAUNCH](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_RETURN_TO_LAUNCH)
+* [MAV_CMD_DO_SET_ROI_WPNEXT_OFFSET](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ROI_WPNEXT_OFFSET)
+* [MAV_CMD_DO_SET_ROI_NONE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ROI_NONE)
+
+
+Note:
+- PX4 parses the above messages, but they are not necessary *acted* on. For example, some messages are vehicle specific.
+- PX4 does not support the frame type [MAV_FRAME_GLOBAL_RELATIVE_ALT_INT](https://mavlink.io/en/messages/common.html#MAV_FRAME_GLOBAL_RELATIVE_ALT_INT)
```
Comment (bkueng) [inline en/flight_modes/mission.md]: Why do you think this is not supported? Generally you can write that local frames are not supported for missions (e.g. MAV_FRAME_LOCAL_NED)

File: en/flying/missions.md [hunk 2]
```diff
@@ -2,22 +2,25 @@
...
 
-![planning-mission](../../images/planning_mission.png)
 
 ## Flying Missions
 
-Once the mission is uploaded, switch to the flight view. It generally
-does not allow to modify the mission, but is optimized to track its
-progress.
+Once the mission is uploaded, switch to the flight view.
+The mission is displayed in a way that makes it easy to track progress (it cannot be modified in this view.
```
Comment (bkueng) [inline en/flying/missions.md]: Missing closing bracket

File: en/flight_modes/mission.md (hunk 1 above)
Comment (hamishwillee) [reply en/flight_modes/mission.md]: It was implied in Dronecode SDK. I will remove for now and put your note in :-)

**Comment by hamishwillee** [comment]
Thanks very much @bkueng . Fixed as you suggested and merged.

**Pull request merged**